packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -894,49 +881,57 @@ class HeroController extends NavigatorObserver {
   void _maybeStartHeroTransition(
     Route<dynamic>? fromRoute,
     Route<dynamic>? toRoute,
-    HeroFlightDirection flightType,
     bool isUserGestureTransition,
```

Comment:
While you're in here: Maybe turn this bool into a (required) named argument to make the call sites clearer.